### ISSUE TITLE
Remove dead code in `twisted/protocols/ftp.py` relating to OTP authentication

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5046]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5046|
|Type|task|
|Created|2011-04-10 15:12:44Z|

Long ago, it seems we tried to implement an OTP scheme for FTP authentication.  This has mostly been removed, but a piece of it remains, the `'userotp'` entry in the `RESPONSE` dict.

<details><summary>Searchable metadata</summary>

```
trac-id__5046 5046
type__task task
reporter__exarkun exarkun
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ftp ftp
keywords__ 
time__1302448364000000 1302448364000000
changetime__1316377824000000 1316377824000000
version__None None
owner__ 
cc__itamarst cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

I have created a branch that tries to fix this ticket.

The branch is here:
https://code.launchpad.net/~adiroiban/twisted/twisted-5046

The LP MP is here:
https://code.launchpad.net/~adiroiban/twisted/twisted-5046/+merge/74920

Cheers,
Adi

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_4"></a>|@itamarst commented|
|-|-|

Thanks for the patch! Note that this would be even easier for me to apply if it included a news file (in this case an empty 5046.misc).

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_5"></a>|@itamarst set status to `closed`|
|-|-|

(In [32545]) Apply patch for 5046.

Author: adiroiban
Fixes: [#5046](https://github.com/twisted/twisted/issues/5046)

Remove unused reference to OTP.
